cmd/topicmappr/commands/rebuild.go

```diff
@@ -43,10 +43,10 @@ func init() {
 	rebuildCmd.Flags().Bool("skip-no-ops", false, "Skip no-op partition assigments")
 	rebuildCmd.Flags().Bool("optimize-leadership", false, "Rebalance all broker leader/follower ratios")
 	rebuildCmd.Flags().Bool("phased-reassignment", false, "Create two-phase output maps")
-
-	// new params for evac leadership
 	rebuildCmd.Flags().String("leader-evac-brokers", "", "Broker list to remove leadership for topics in leader-evac-topics.")
 	rebuildCmd.Flags().String("leader-evac-topics", "", "Topics list to remove leadership for the brokers given in leader-evac-brokers")
+	rebuildCmd.Flags().Bool("chunked", false, "Output the result map in a series of smaller sequential maps that work towards the desired state.")
+	rebuildCmd.Flags().Int("chunk-step-size", 3, "Number of brokers to move data at a time for with a getPartitionMapChunk operation.")
```

Comment:
nit: maybe set chunk-step-size to 0 by default and remove the chunked switch to only consider chunked when chunk-step-size >= 1